### ISSUE TITLE
Allow indent inside query blocks

### ISSUE THREAD

**Issue by hacknug** [gridsome/eslint-plugin-gridsome#39]
**Please write rule name what you want to improve:** `format-query-block`

**How are the rules improved?:** Rules should allow queries to be indented one level inside their block.

**Provide 2-3 code examples that this rule should warn about:**
This is valid right now:
```vue
<static-query>
query {
  metadata {
    siteName
  }
}
</static-query>
```

This is valid NOT right now:
```vue
<static-query>
  query {
    metadata {
      siteName
    }
  }
</static-query>
```

**Additional context**
VSCode allows you to collapse/fold blocks based on indentation. If this isn't allowed, anything inside `static-query` or `page-query` will be visible.

**Comment by tyankatsu0105** [comment]
@hacknug 
Thanks for the issue!

>allow queries to be indented one level inside their block.

I like this. I'll try doing that.

**Comment by tyankatsu0105** [comment]
Hey @hacknug👋
I released eslint-plugin-gridsome v1.4.0 just now!!
This rule supports top level indent in <page-query> and <static-query>.
Indent follow your prettierrc's option `useTabs` and `tabWidth`.

Thank you again that sending this issue.🙏

**Comment by tyankatsu0105** [comment]
This is demo💁‍♂️
![demo](https://user-images.githubusercontent.com/28397593/68082861-3ec8a680-fe65-11e9-9a04-06ed6dbc1d72.gif)

**Comment by hacknug** [comment]
Awesome! Just tried it and it works like a charm. Thank you ❤️
